src/components/NavBar/NavBar.jsx

```diff
@@ -38,7 +38,7 @@ const NavBar = () => {
   const [openMenu, setOpenMenu] = useState(false);
   const menuId = 'primary-search-account-menu';
 
-  // state for LogoutModal component
+  // state for Modal component
```

Comment:
As a dev, if we just change this - again, I'd be asking "what's this modal for?" Then have to dig around the code and figure it out.  The only way to figure it out is looking at the import statement up top because I have the file name for the component as `LogoutModal`, but if it wasn't specific then we'd have to go to that file and read the code to figure it out.

Eliminating 5 characters here from the code base actually can result in a lot of dev wasted time (when you think about all the new volunteers that come in and out of this project.) Not a good tradeoff for a very minimal amount of "extra" code.

Reply:
I really don't know, may it occur due because of cloning it instead of a fork,